.azure-pipelines.yml

```diff
@@ -127,6 +134,39 @@ jobs:
     parameters:
       directory: tools/wpt/
 
+- job: infrastructure_win10
+  displayName: 'infrastructure/ tests (Windows 10)'
+  #dependsOn: decision
+  #condition: dependencies.decision.outputs['test_jobs.wptrunner_infrastructure']
+  pool:
+    name: 'Hosted Windows Client'
+  steps:
+  - template: tools/ci/azure/system_info.yml
+  - template: tools/ci/azure/checkout.yml
+  - template: tools/ci/azure/install_python.yml
+  - template: tools/ci/azure/pip_install.yml
+    parameters:
+      packages: virtualenv
+  - template: tools/ci/azure/install_certs.yml
+  - template: tools/ci/azure/update_hosts.yml
+  - template: tools/ci/azure/update_manifest.yml
+  - script: python ./wpt run --yes --no-manifest-update --install-fonts --manifest MANIFEST.json --metadata infrastructure/metadata/ --log-tbpl $(Build.ArtifactStagingDirectory)/edge.tbpl.log --log-tbpl-level info edge_webdriver infrastructure/
+    displayName: 'Run tests (Edge)'
+  - task: PublishBuildArtifacts@1
+    displayName: 'Publish results'
+    inputs:
+      artifactName: 'infrastructure'
+    condition: succeededOrFailed()
+  - powershell: |
```

Comment:
Why is this not in a YML file like the hosts update?

Reply:
I don't like this bit, hoping to not need it when this is merged. That's blocked on the agent pool being reliably populated with responsive agents that don't get their state messed up so easily.

Reply:
What else is this waiting on now? You've got the review you asked for now. :)

Reply:
I'll write email about the agent pool.